daq/faucet_state_collector.py

```diff
@@ -369,13 +374,26 @@ def process_dp_config_change(self, timestamp, dp_name, restart_type, dp_id):
                 return
 
             dp_state = self.switch_states.setdefault(dp_name, {})
-            dp_state = self.switch_states.setdefault(dp_name, {})
 
             dp_state[KEY_DP_ID] = dp_id
             dp_state[KEY_CONFIG_CHANGE_TYPE] = restart_type
             dp_state[KEY_CONFIG_CHANGE_TS] = datetime.fromtimestamp(timestamp).isoformat()
             dp_state[KEY_CONFIG_CHANGE_COUNT] = dp_state.setdefault(KEY_CONFIG_CHANGE_COUNT, 0) + 1
 
+    @dump_states
+    def process_dp_change(self, timestamp, dp_name, connected):
+        """process dp_change to get dp status"""
+        with self.lock:
+            if not dp_name:
+                return
+            dp_state = self.switch_states.setdefault(dp_name, {})
+            #using "CONNECTED" as placeholder until we figure out distinction b/w HEALTHY and DAMAGED
```

Comment:
Nit: 

# Using

Reply:
That was supposed to be "# Using" rather than "#using"

-- markdown messed it up!

Reply:
Actually, should be:
# TODO: Figure out... 
or something like that.
"TODO" are special and some tools flag them...